_data/hosting_providers.json

```
@@ -1929,6 +1929,16 @@
     "reviewed": "2019.6.6",
     "note": ""
   },
+  {
+    "name": "Vimexx",
+    "link": "",
+    "category": "partial",
+    "tutorial": "https://www.vimexx.nl/help/gratis-lets-encrypt-ssl-op-je-website-installeren",
+    "announcement": "https://www.vimexx.nl/news/gratis-wildcard-ssl-via-lets-encrypt",
+    "plan": "",
+    "reviewed": "2021.2.28",
+    "note": ""
```

Although it hasn't been added consistently in every entry, I think it's valuable to include a note about the language:

```suggestion
    "note": "The website is in Dutch."
```